spec/support/mockResponseData.json

```diff
@@ -1,5 +1,180 @@
 {
-    "bid": [],
-    "pass": [],
-    "deals": []
-}
\ No newline at end of file
+    "bids": [
```

Comment:
As a double check, do `bids`, `pass` and `deals` always exist in the response? Can they co-exist in the same response?